File: src/go/chi/models/complex_result.go
package models

type ComplexQueryResult struct {
	PeriodDays int           `json:"period_days"`
	TotalUsers int           `json:"total_users"`
	Data       []UserStats   `json:"data"`
}

type UserStats struct {
	UserID     int     `json:"user_id"`
	UserName   string  `json:"user_name"`
	TotalOrders int    `json:"total_orders"`
	TotalValue  float64 `json:"total_value"`
	AverageValue float64 `json:"average_value"`
}

File: src/kotlin/spring/src/main/kotlin/com/benchmark/model/ComplexResult.kt
package com.benchmark.model

data class ComplexResult(
    val userId: Int,
    val userName: String,
    val totalOrders: Long,
    val totalValue: Double,
    val averageOrderValue: Double
)

data class ComplexResponse(
    val periodDays: Int,
    val totalUsers: Int,
    val data: List<ComplexResult>
)
